arbor/communication/communicator.cpp

```diff
@@ -104,7 +104,7 @@ communicator::communicator(const recipe& rec,
     // The loop above gave the information required to construct in place
     // the connections as partitioned by the domain of their source gid.
     connections_.resize(n_cons);
-    connection_part_ = algorithms::make_index(src_counts);
+    connection_part_ = util::make_index(src_counts);
```

Comment:
`util::make_partition` is a possible alternative here (from `util/partition.hpp`).